dropbox/.env-template

```diff
@@ -3,4 +3,8 @@ DROPBOX_APP_KEY=
 DROPBOX_APP_SECRET=
```

Comment:
The variables DROPBOX_APP_KEY and DROPBOX_APP_SECRET are not used. Let's remove them here and add notes in the README related to the connector registration block

Reply:
Done - removed, there is a section in the README that requires the app secret and key